src/features/metadata-instance-editor/TemplateDropdown.js

```diff
@@ -38,15 +38,6 @@ type State = {
     templates: Array<MetadataTemplate>,
 };
 
-const InputContainer = ({ inputProps = {}, ...rest }: { inputProps?: Object }) => (
```

Comment:
I removed this because it didn't seem to serve any purpose being wrapped inside of a container/functional component